docs/NMOS With JPEG XS.md

```diff
@@ -23,20 +23,45 @@ and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119]
-This has been permitted since IS-04 v1.1.
+Nodes capable of transmitting JPEG XS video streams MUST have Source, Flow and Sender resources in the IS-04 Node API.
 
-Nodes implementing [BCP-004-01][] Receiver Capabilities use the existing `constraint_sets` parameter within the `caps` object, describing combinations of frame rates, width and height, and other parameters which the receiver can support, using the parameter constraints defined in the [Capabilities register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/) of the [NMOS Parameter Registers][].
+### Sources
 
-If the Receiver supports streams meeting the traffic shaping and delivery timing requirements for ST 2110-22, it SHOULD use the `urn:x-nmos:cap:transport:st2110_21_sender_type` parameter constraint.
+The Source resource MUST indicate `urn:x-nmos:format:video` for the `format`.
+Source resources can be associated with many Flows at the same time.
+The Source is therefore unaffected by the use of JPEG XS compression.
 
-An example Receiver resource is provided in the [Examples](../examples/).
+### Flows
 
-## JPEG XS IS-04 Senders, Flows and Sources
+The Flow resource MUST indicate `video/jxsv` in the `media_type` attribute, and `urn:x-nmos:format:video` for the `format`.
+This has been permitted since IS-04 v1.1.
 
-Nodes capable of transmitting JPEG XS video streams MUST have Sender, Flow and Source resources in the IS-04 Node API.
+For Nodes implementing IS-04 v1.3 or higher, the following additional requirements on the Flow resource apply.
+
+In addition to those attributes defined in IS-04 for all coded video Flows, the following attributes defined in the [Flow Attributes register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/) of the [NMOS Parameter Registers][] are used for JPEG XS.
+
+- [Components](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#components)  
+  The Flow resource MUST indicate the color (sub-)sampling using the `components` attribute.
+  The `components` array value corresponds to the `sampling`, `width` and `height` values in the SDP format-specific parameters defined by RFC 9134.
+- [Profile](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#profile)  
+  The Flow resource MUST indicate the JPEG XS profile, which defines limits on the required algorithmic features and parameter ranges used in the codestream.
+  The permitted `profile` values are strings, defined as per RFC 9134.
+- [Level](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#level)  
+  The Flow resource MUST indicate the JPEG XS level, which defines a lower bound on the required throughput for a decoder in the image (or decoded) domain.
+  The permitted `level` values are strings, defined as per RFC 9134.
```

Comment:
@tbr Would this be better? Or did we decide to reject describing Unrestricted profile/level?

```suggestion
- [Profile](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#profile)  
  The JPEG XS profile defines limits on the required algorithmic features and parameter ranges used in the codestream.
  When the `profile` attribute is omitted, the Unrestricted profile is implied.
  Otherwise, the `profile` value MUST be a string, defined as per RFC 9134, such as `High444.12`.
- [Level](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#level)  
  The JPEG XS level defines a lower bound on the required throughput for a decoder in the image (or decoded) domain.
  When the `level` attribute is omitted, the Unrestricted level is implied.
  Otherwise, the `level` values MUST be a string, defined as per RFC 9134, such as `1k-1`.
```

Reply:
I guess you could write: When the `level` attribute is omitted, the level is unknown and shall be specified by the codestream.

This is how the RFC and SDP also interpret it.

Reply:
Although then we don't have an explicit way to signal Unrestricted...

Reply:
Not explicitly no... but one could say that omitting is equally difficult to signaling `Unrestricted` which not many implementations will handle (such decoder would need to start decoding and hope for the best, so I don;t see this as a practical approach in the field).